docs/UserGuide.md

```diff
@@ -87,132 +87,170 @@ Shows a message explaining how to access the help page.
-* `START_TIME` is strictly less than `END_TIME`.
-* Clashing activity (including partial overlap with other activities or appointments) would be considered as invalid
-  activity (i.e. not to be added).
-* `TITLE` ideally should be very short so that it can be displayed in the list without being truncated.
-
-**Tip:** Try to pack as many keywords as possible inside the description while keep it concise so that you can search it
-later.
+* `NAME` indicates the full name of the patient, first name and last name are separated by `-`.
+* `AGE` is greater than or equal to 0.
+* `GENDER` is chosen from one of 3 choices, `M/F/O` where `M` is for Male, `F` is for Female, and `O` is for Others.
+* `HEIGHT` is the height of patient in centimeters rounded to the nearest integer.
+* `WEIGHT` is the weight of patient in kilograms rounded to the nearest integer.
+* `BLOOD_TYPE` is chosen from one of the 4 choices, `A/B/AB/O`.
+* `CONTACT_NUMBER` must be 8-digit Singapore phone number.
+* `OPTIONAL` is the list of patient's past/pre-existing medical conditions. For medical condition that has multiple
+words, use `-` to combine the words, e.g. `heart-failure`. To separate between conditions, use `/`.
 
 Examples:
 
-* `add t/activity s/2022-09-15-14-00 e/15/09/2022 15:00 ti/Meeting with Mr. X d/about a certain paper`
-* `add t/activity s/15/09/2022 14:00 e/15/09/2022 15:00 ti/Meeting with Mr. Y`
+* `add p/patient n/John-Doe a/30 g/M h/174 w/72 b/O c/12345678 o/heart-failure/diabetes`
+* `add p/patient n/Tom-Doe a/20 g/M h/167 w/61 b/AB c/12341234`
 
-### Deleting an existing activity: `delete t/activity`
+### Deleting an existing patient: `delete p/patient`
 
-Delete a certain existing activity
+Deletes a patient from the GoMedic application.
 
-Format: `delete t/activity i/ACTIVITY_ID`
+Format: `delete p/patient i/PATIENT_ID`
 
-* Activity ID can be obtained by listing all the activities or search for a certain activities within a certain time
-  frame.
-* Activity ID is **unique** (i.e. every activity will be assigned to a unique ID, hence this guarantees
-  1 `delete t/activity` command will not delete 2 activities at once).
-* Invalid Activity ID being supplied would be flagged by GoMedic, and do not cause changes to any existing activities.
+* Patient ID can be obtained by listing all the patients or search for a certain patients with available filters.
+* Patient ID is **unique** (i.e. every patient will be assigned to a unique ID, hence this guarantees
+  1 `delete p/patient` command will not delete 2 patients at once).
+* Invalid Patient ID being supplied would be flagged by GoMedic, and do not cause changes to any existing patients.
 
 Examples:
 
-* `delete t/activity A123`
+* `delete p/patient i/P001`
+
+### Updating an existing patient: `update p/patient`
+
+Updates a patient's details from the GoMedic application.
+
+Format: 
+`update p/patient i/PATIENT_ID n/NAME a/AGE g/GENDER h/HEIGHT w/WEIGHT b/BLOOD_TYPE c/CONTACT_NUMBER
```

Comment:
I believe there are notations for optional flag, maybe can try to change the existing update and use the `[ ]` notation, because this format means patient id, name, age etc are compulsory